dashboard/test/factories.rb

```diff
@@ -502,6 +507,15 @@
     workshop_type Pd::Workshop::TYPES.first
     course Pd::Workshop::COURSES.first
     capacity 10
+    transient do
+      num_sessions 0
+    end
+    after(:create) do |workshop, evaluator|
+      # Sessions, one per day starting today
```

Comment:
It doesn't matter in this context. I just wanted a factory to create `n` sessions, and they must be on different days based on the model validations. None of the tests care which specific day they are on, and if we ever need that we can use the existing `pd_session` factory to create them directly with a set date.